integration_tests/src/lib.rs

```diff
@@ -2657,6 +2667,9 @@ mod test {
         // Clone a read only downstairs to the new downstairs
         // Make a new volume with two old and the one new downstairs and verify
         // the original data we wrote.
+        // Create a new downstairs directly from the clone.
```

Comment:
pasted from somewhere else?

Reply:
No, that I really added.
This test did it the old way, where it made the region first, then did the clone.
I added this section that does the create+clone in one step.
(then does the same test we did for the "create then clone" test.)

Reply:
Ah, I get it now - these are steps the test takes